python/python_fastapi_server/configure_logging.py

```diff
@@ -3,7 +3,17 @@
 """Configures logging for the adaguc-server python wrapper"""
 import sys
 
+def run_once(f):
+    def wrapper(*args, **kwargs):
+        if not wrapper.has_run:
+            wrapper.has_run = True
```

Comment:
Change the order like [here](https://gist.github.com/jasongrout/3804691).
If `configure_logging` somehow fails, it will not run again, as `has_run` is set to True before it is run.

Reply:
Okay, I think I added the @wraps correctly.

Reply:
Now completely removed the run_once part.